src/filesystem/interface.js

```diff
@@ -159,6 +160,27 @@ function FileSystem(options, callback) {
     return watcher;
   };
 
+  this.unwatchFile = function(filename, listener) {
+    if(isNullPath(filename)) {
+      throw new Error('Path must be a string without null bytes.');
+    }
+    listener = listener || nop;
+
+    if(listener == nop){
+      this.removeAllListeners();
+    }
+    else{
+      this.off('change', listener);
```

Comment:
For what you're trying to do to work, we're going to have to alter how `.watch()` works above, namely, we're going to need to cache the listener for the given filename on an `Object` accessible by both of these functions, and keyed on the filename, such that we can access it here.

Probably on line 159 above, when you have both the `filename`, the `watcher`, and the `listener`, you'll need to add it to an `Object` that we define somewhere, either in this file on line 143, or maybe we do it in `FSWatcher` itself.

Probably the latter is best, since we're normalizing the filename there.  Over [here](https://github.com/filerjs/filer/blob/db315eae134c1e0d9d9b47531615b13c4abb9026/src/fs-watcher.js#L61) you need something like this:

```js
// At the top of the file
var _watchers = {};

FSWatcher.getWatcherForFilename = function(filename) {
  return _watchers[filename];
};
```

Now in the code for `start()`, you can cache the `watcher` instance in `_watchers`, maybe [here](https://github.com/filerjs/filer/blob/db315eae134c1e0d9d9b47531615b13c4abb9026/src/fs-watcher.js#L48):

```js
_watchers[filename] = self;
```
Now you have what you need to call `.close()` here when you want to remove this watcher.

Do you follow what I'm saying?